R/api.R

```diff
@@ -39,7 +39,7 @@ crunchAPI <- function(
 #' These methods let you communicate with the Crunch API, for more background
 #' see [Crunch Internals](https://crunch.io/r/crunch/articles/crunch-internals.html).
 #'
-#' @param ... see [`crunchAPI`] for details. `url` is the first
+#' @param url,config,body,... see [`crunchAPI`] for details. `url` is the first
```

Comment:
The help page now looks a bit odd (compare GET/DELETE & PUT/PATCH/POST, they all get url/config, but it's explicit only in the latter group).

Reply:
Made them consistent at least